Resources/Prototypes/Entities/Objects/Power/lights.yml

```diff
@@ -95,16 +94,53 @@
 - type: entity
   parent: BaseLightbulb
   name: led light tube
+  description: A high power high energy bulb.
   id: LedLightTube
   components:
   - type: LightBulb
     bulb: Tube
     color: "#EEEEFF"
-    lightEnergy: 4
+    lightEnergy: 2.5
     lightRadius: 10
     lightSoftness: 0.9
     BurningTemperature: 350
-    PowerUse: 9
+    PowerUse: 25
+  - type: Sprite
+    sprite:  Objects/Power/light_tube.rsi
+    state: normal
+
+- type: entity
+  parent: BaseLightbulb
+  name: exterior light tube
+  description: A high power high energy bulb for the depths of space. May contain mercury.
+  id: ExteriorLightTube
+  components:
+  - type: LightBulb
+    bulb: Tube
+    color: "#9cbcff"
```

Comment:
I would also like to propose `#B4FCF0`.
![image](https://user-images.githubusercontent.com/7548248/150980240-c78578ea-9cc4-4257-8902-b64074f9de7c.png)

Reply:
Done :)